footer.php

```diff
@@ -130,6 +132,49 @@
 	});
 </script>
 
+<script type="text/javascript">
+
+	$( '.link-primary' ).bind( "mouseenter", function(e) {	
+		$( '.link-primary' ).removeClass( 'open' );
+		$(this).find( '.main-nav-link' ).attr('aria-expanded', 'true');
+		$(this).closest( '.link-primary' ).addClass( 'open' );
+	});
+	$( '.link-primary' ).bind( "mouseleave", function(e) {	
+		$(this).find( '.main-nav-link' ).attr('aria-expanded', 'false');
+		$( '.link-primary' ).removeClass( 'open' );
+	});
+
+
+	$('#nav-main').keydown(function(e) {
+        var $focused = document.activeElement;
+
+        switch(e.keyCode)
+        {
+            // user presses the "space"
+            case 32:    showMenu(e, $focused);
+                        break;
+
+            // user presses the "esc" key
+            case 27:    hideMenu(e,);
```

Comment:
This line is throwing a syntax error in Firefox and IE on Windows. `expected expression, got ')'`